docs/guides/advanced/timezones.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Timezones
+---
+
+# Timezones in Coda
+
+Every Coda document has a **Timezone** setting in the **Region** section of the settings menu, that defaults to the timezone of the device that created it. All dates and times entered in the document are interpreted in the context of that timezone. Users often don't need to be aware this, since all of their interactions with the dates and times take place in the context of that doc.
+
+Packs don't have a configurable timezone however, and all date and time operations in Pack code run in [Coordinated Universal Time (UTC)][wikipedia_utc]. The timezone of the document is passed to Pack formulas as [`context.timezone`][ExecutionContext_timezone], containing an [IANA timezone identifier][wikipedia_tz] (ex: `America/New_York`).
+
+
+## Timezone shifting {: .shifting}
+
+Any date or time values passed into a Pack are shifted from the doc's timezone to UTC. For example, the value "11/12/1955, 10:04 PM" in a document with the timezone set to "America/Los Angeles" will be passed to a Pack as `1955-11-13T06:04:00.000Z`. While this accurately represents the exact same moment in time, you'll notice that the day and hour values have changed due to the change in timezone.
```

Comment:
lol I like your example of using 1955. lol

Reply:
Back to the Future reference!